apps/condo/domains/news/tasks/notifyResidentsAboutNewsItem.spec.js

```diff
@@ -26,6 +27,7 @@ const { makeClientWithResidentUser } = require('@condo/domains/user/utils/testSc
 
 describe('notifyResidentsAboutNewsItem', () => {
     setFakeClientMode(index)
+    jest.setTimeout(30000)
```

Comment:
it's needed here?
if yes, don't it's need to be reset after this block?

Reply:
removed